packages/ffe-buttons/less/inline-base-button.less

```diff
@@ -71,10 +71,8 @@
     color: @ffe-blue-azure;
     flex-wrap: wrap;
 
-    &::after {
+    .ffe-inline-button__label {
```

Comment:
There was some reason I decided to solve this by means of a pseudo element - but I can't for the life of me remember what it was.